package.js

```diff
@@ -1,13 +1,13 @@
 Package.describe({
   name: "dburles:collection-helpers",
   summary: "Transform your collections with helpers that you define",
-  version: "1.1.0",
+  version: "1.2.0",
   documentation: "README.md",
   git: "https://github.com/dburles/meteor-collection-helpers.git",
 });
 
 Package.onUse(function(api) {
-  api.versionsFrom('1.4.2');
+  api.versionsFrom(['1.12.1', '2.3', '3.0']);
```

Comment:
Hey @drone1 just wondering why the minimum version bump here is required?

Reply:
Sorry, actually it looks like you're referring to the package minimum version but you replied on this line about minimum Meteor versions. If so, I bumped the package version since there is an update? Sorry I'm not understanding the question. Thank you.

Reply:
What is the specific trouble you were having running the current version of the package on Meteor 3? It works okay for me.

Reply:
What I meant here is that you have dropped support for Meteor versions between 1.4.2 and 1.12.1 and I was wondering if there was any specific reason for that?

Reply:
> What is the specific trouble you were having running the current version of the package on Meteor 3? It works okay for me.

Helpers weren't working on a client collection on document retrieved with `findOne()`. Once I changed the `api.versionFrom` line everything worked again from what I recall, but let me confirm and get back here with details.

> What I meant here is that you have dropped support for Meteor versions between 1.4.2 and 1.12.1 and I was wondering if there was any specific reason for that?

This was totally copy-pasta :( Will fix. Thanks.

Reply:
I can't reproduce the problem you mention, maybe something else is going on?